src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -747,21 +786,60 @@ def __call__(
         # 4. Preprocess image
         image = self.image_processor.preprocess(image)
 
-        # 5. set timesteps
+        # 5. Prepare timesteps
+        def denoising_value_valid(dnv):
+            return type(denoising_end) == float and 0 < dnv < 1
```

Comment:
Is this what you want here instead?
```suggestion
            return type(dvd) == float and 0 < dnv < 1
```